patches/dbus_activation_service.py.patch

```diff
@@ -0,0 +1,82 @@
+--- /dev/null	2019-11-07 08:53:04.471987251 -0800
++++ kolibri-0.12.8/kolibri/core/content/management/commands/dbus_activation_service.py	2019-11-08 16:34:21.992473978 -0800
+@@ -0,0 +1,79 @@
++import dbus
```

Comment:
Assuming you wrote this, `python-dbus` is dead, don't use it. Use `Gio` via pygobject that provides DBus support:

- http://lazka.github.io/pgi-docs/#Gio-2.0/classes/DBusProxy.html#Gio.DBusProxy